### PR TITLE
Add location to profiles

### PR DESCRIPTION
**Pull request by andrewda**
*mentioned in https://github.com/gitpoint/git-point/issues/17#issuecomment-314650595*

User profile:

screen shot 2017-07-17 at 4 36 47 pm

Organization:

screen shot 2017-07-17 at 4 36 38 pm

### CODE REVIEW

**Comment by andrewda** [comment]
Yep good idea. That'd make everything more consistent!

**Comment by andrewda** [comment]
Look good?

screen shot 2017-07-17 at 11 32 02 pm
screen shot 2017-07-17 at 11 32 40 pm

File: src/components/entity-info.component.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  },
+});
+
+const getBlogLink = url =>
+  url.substr(0, 4) === 'http' ? url : `http://${url}`;
+
+const getLocationLink = location =>
+  `https://www.google.com/maps/place/${location.split(' ').join('+')}`;
+
+export const EntityInfo = ({ entity }: Props) => {
```
Comment (lex111) [inline src/components/entity-info.component.js]: May be better called `UserInfo`? Information applies to the user.

**Comment by housseindjirdeh** [comment]
I think that looks perfect 🎉 I like the title being just 'Info', @lex111 we can discuss changing it afterwards if you like

**Pull request merged**